Makefile

```diff
@@ -1,4 +1,4 @@
-EXTVERSION   = dev
```

Comment:
Is this a number that will have to be bumped with every release?

Reply:
Sorry I meant to comment on the other side of the diff.

Reply:
It looks like it has to be bumped, unfortunately. Which is why I've reverted to that.

Reply:
Okay. Cool.